src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/SpecimenCharacteristicFastingStatus.java

```diff
@@ -1,44 +0,0 @@
-package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
-
-import lombok.EqualsAndHashCode;
-import lombok.Getter;
-import lombok.RequiredArgsConstructor;
-import org.apache.commons.lang3.StringUtils;
-import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.EdifactValidationException;
-import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.Split;
-
-/**
- * Example SPC+FS+F'
- */
-@RequiredArgsConstructor
-@Getter
-@EqualsAndHashCode(callSuper = false)
-public class SpecimenCharacteristicFastingStatus extends Segment {
```

Comment:
So is this no longer needed? I can see in the docs the fasting status is optional

Reply:
@matthewdoreilly found that it was actually from a newer version than we're using, so it is not something we're supporting.